app/components/modals/ChangePassphraseModal/ChangePassphraseModal.jsx

```diff
@@ -0,0 +1,32 @@
+import Modal from "./ChangePassphraseModalContent";
+import useChangePassphraseModal from "./hooks";
+
+function ChangePassphraseModal({ onCancelModal, onSubmit, ...props }) {
+  const {
+    onCancelModalCallback,
+    validationFailed,
+    isValid,
+    onSubmitCallback,
+    updatePrivatePassphrase,
+    updateConfirmPrivatePassphrase,
+    onTriggerPassphraseModalSubmit,
+    ...state
```

Comment:
This is repeating in all modals, is it needed? the hooks seem to return all the values, so not sure if this needed.

Reply:
The ``state`` variables are the hook states used in ``***Content.jsx`` components and/or their children. The unused states in their children components aren't being returned from ``hooks.js``.  It can be changed if returned values from hooks are declared and passed as props explicitly as well. I'm open to suggestions here.

Reply:
I prefer declare and explicit call them, because it makes it easier to know what props a component is using. Also, with hooks there are no more states declaration, so I think it may be confunsing.